Python 3.6.5 (default, Apr  1 2018, 05:46:30) 
[GCC 7.3.0] on linux
Type "copyright", "credits" or "license()" for more information.
>>> 
============= RESTART: /home/user/RTR105/sin_caur_summu_ver5.py =============
Lietotāj, lūdzu, iedavi argumentu (x): 1.89
Traceback (most recent call last):
  File "/home/user/RTR105/sin_caur_summu_ver5.py", line 6, in <module>
    print("sin(%.2f) = %6.2f"%(x,y))
TypeError: must be real number, not builtin_function_or_method
>>> 
============= RESTART: /home/user/RTR105/sin_caur_summu_ver5.py =============
Lietotāj, lūdzu, iedavi argumentu (x): 1.89
Traceback (most recent call last):
  File "/home/user/RTR105/sin_caur_summu_ver5.py", line 6, in <module>
    print("sin(%.2f) = %6.2f"(x,y))
TypeError: 'str' object is not callable
>>> 
============= RESTART: /home/user/RTR105/sin_caur_summu_ver5.py =============
Lietotāj, lūdzu, iedavi argumentu (x): 1.89
Traceback (most recent call last):
  File "/home/user/RTR105/sin_caur_summu_ver5.py", line 6, in <module>
    print("sin(%.2f) = %6.2f"(x,y))
TypeError: 'str' object is not callable
>>> 
============= RESTART: /home/user/RTR105/sin_caur_summu_ver5.py =============
Lietotāj, lūdzu, iedavi argumentu (x): 1.89
Traceback (most recent call last):
  File "/home/user/RTR105/sin_caur_summu_ver5.py", line 6, in <module>
    print("sin(%.2f) = %6.2f"(x,y))
TypeError: 'str' object is not callable
>>> 
============= RESTART: /home/user/RTR105/sin_caur_summu_ver5.py =============
Lietotāj, lūdzu, iedavi argumentu (x): 1.89
sin(1.89) =   0.95
a0 =   1.89 S0 =   1.89
a1 = -10.13 S1 =  -8.24
a2 =  45.22 S2 =  36.98
a3 = -188.44 S3 = -151.46
>>> 
============= RESTART: /home/user/RTR105/sin_caur_summu_ver5.py =============
>>> 
Lietotāj, lūdzu, iedavi argumentu (x): 1.5
sin(1.50) =   1.00
a0 =   1.50 S0 =   1.50
a1 =  -5.06 S1 =  -3.56
a2 =  14.24 S2 =  10.68
a3 = -37.38 S3 = -26.70
>>> 
